synapse_rock/rockcraft.yaml

```diff
@@ -16,37 +16,38 @@ parts:
     synapse:
         build-packages:
             - build-essential
+            - ca-certificates
```

Comment:
Should probably be installed from `overlay-packages` because ca-certificates relies a lot of post-script from the APT package which aren't executed in `build-packages` or `stage-packages` steps but are in `overlay-packages`
See https://discourse.canonical.com/t/eod-digest/2098/215?u=gschiano

This will probably allow you to remove the `cp` at the end of `override-build`

Reply:
Awesome! Thanks for the tip, changed.